crates/nu-utils/src/sample_config/default_config.nu

```diff
@@ -210,67 +215,38 @@ $env.config = {
   # datetime_format determines what a datetime rendered in the shell would look like.
   # Behavior without this configuration point will be to "humanize" the datetime display,
   # showing something like "a day ago."
-
   datetime_format: {
     normal: '%a, %d %b %Y %H:%M:%S %z'  # shows up in displays of variables or other datetime's outside of tables
-    # table: '%m/%d/%y %I:%M:%S%p'        # generally shows up in tabular outputs such as ls. commenting this out will change it to the default human readable datetime format
+    table: '%m/%d/%y %I:%M:%S%p'        # generally shows up in tabular outputs such as ls. commenting this out will change it to the default human readable datetime format
```

Comment:
I don't agree that this should be the default for nushell tables. I think we should have the human readable date as the default as it's been in nushell since day one. So, I think we need to comment this out.

Reply:
mm yeah :thinking: 

the annoying is that there is no "*humanized*" format for datetimes...

`date now` does not give the same result by default and with `$env.config.datetime_format.normal` set :confused:

Reply:
as we do not have a *human* format for datetimes, we can not define `datetime_format`s that are the exact same as without any config, so i've commented out both the fields in 0ced6b8ec :relieved:

Reply:
if you comment out the table line, like it was before, it prints out in human readable form

Reply:
> if you comment out the table line, like it was before, it prints out in human readable form
>
yes, this is what i've done in 0ced6b8 :wink: